client/dashboard/sites/overview-domains-card/index.tsx

```diff
@@ -34,5 +129,15 @@ export default function DomainsCard( { site }: { site: Site } ) {
 		return <DomainUpsellCard site={ site } />;
 	}
 
-	return null;
+	if ( isDashboardBackport() ) {
```

Comment:
I decided not to backport the Domain names card since it's too dependent on v2 routes and adding a bunch of `isDashboardBackport()` logic doesn't seem worth it. Especially when v1 dashboard doesn't have the site-level domain tab, so the "Manage Domains" button could only link to the Calypso admin.

Reply:
This check needs to be before the `return <CalloutSkeleton>` check; otherwise v1 will render a skeleton which just disappears after a while, which is confusing.